adoc/Private-Registry-Administration.adoc

```diff
@@ -743,3 +743,206 @@ suse-registry-harbor-registry-65854df7bc-mrfnj     2/2     Running   0
+Labels:       velero.io/storage-location=default
+Annotations:  <none>
+
+Phase:  Completed
+
+Namespaces:
+  Included:  registry
+  Excluded:  <none>
+
+Resources:
+  Included:        *
+  Excluded:        <none>
+  Cluster-scoped:  auto
+
+Label selector:  <none>
+
+Storage Location:  default
+
+Snapshot PVs:  auto
+
+TTL:  720h0m0s
+
+Hooks:  <none>
+
+Backup Format Version:  1
+
+Started:    2020-12-17 12:00:12 +0100 CET
+Completed:  2020-12-17 12:01:36 +0100 CET
+
+Expiration:  2021-01-16 12:00:12 +0100 CET
+
+Persistent Volumes: <none included>
+
+Restic Backups (specify --details for more information):
+  Completed:  4
+----
+
+.Backup options
+[NOTE]
+====
+For more advanced options, such as scheduling backups and configuring its expiration time, check the link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[Velero documentation].
+====
+
+===== Restore
+
+. If the restoration is performed on a new Kubernetes cluster, install Velero by following the link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[documentation]. Make sure to use the same storage backend used to perform the backup, so that Velero can access the backups performed previously. If restoring on the same cluster, make sure to completely delete the existing {spr} deployment, including its namespace.
+
+. Perform the restore by running the command `velero restore create <restore-name> --from-backup <backup-name>`, for example:
++
+[source,bash]
+----
+> velero restore create registry-restore --from-backup registry-backup
+Restore request "registry-restore" submitted successfully.
+Run `velero restore describe registry-restore` or `velero restore logs registry-restore` for more details.
+----
+
+. Run `velero restore describe <restore-name>` to check the restore status and ensure its completion. For example:
++
+[source,bash]
+----
+> velero restore describe registry-restore
+Name:         registry-restore
+Namespace:    velero
+Labels:       <none>
+Annotations:  <none>
+
+Phase:  Completed
+
+Errors:
+  Velero:     <none>
+  Namespaces: <none>
+
+Backup:  registry-backup
+
+Namespaces:
+  Included:  *
+  Excluded:  <none>
+
+Resources:
+  Included:        *
+  Excluded:        nodes, events, events.events.k8s.io, backups.velero.io, restores.velero.io, resticrepositories.velero.io
+  Cluster-scoped:  auto
+
+Namespace mappings:  <none>
+
+Label selector:  <none>
+
+Restore PVs:  auto
+
+Restic Restores (specify --details for more information):
+  Completed:  4
+----
+
+.Restore options
+[NOTE]
+====
+For more advanced options, such restoring from an scheduled backup, see the link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[Velero documentation].
+====
+
+==== Backup and restore of external services
+
+For the backup and restore of external services, see the official documentation of the adopted solution. For example:
```

Comment:
Either stick to the fairly common, standard, "backup and restore", _or_ the more grammatically-correct "backup and restoration", but do not mix both. I leave it up to you which one you would prefer.

Reply:
I prefer to use "backup and restore" so we keep it consistent with the Velero documentation.